doc/guide/dynamics/dynamics-monte.rst

```diff
@@ -89,44 +92,80 @@ To illustrate the use of the Monte Carlo evolution of quantum systems in QuTiP,
+The main difference with :func:`qutip.mesolve`'s :class:`qutip.Result` is that it optionally store the result of each trajectories with the averages.
+When trajectories are stored, ``result.runs_expect`` is a list over the expectation operators, trajectories and times in that order.
+The averages are stored in ``result.average_expect`` and the standard derivation of the expectation values in ``result.std_expect``.
+When states are returned, ``result.runs_states`` will be an array of length ``ntraj``, with each element containing an array of ket-type qobjs with the same number of elements as ``times`` and ``result.average_states`` is a list of density matrices for each times.
+Furthermore, the output will also contain a list of times at which collapse occurred, and which collapse operators did the collapse, in the ``col_times`` and ``col_which`` properties, respectively.
+Lastly ``result.photocurrent`` contain the measurement of the evolution.
 
-As mentioned earlier, by default, the ``mcsolve`` function runs 500 trajectories.  This value was chosen because it gives good accuracy, Monte Carlo errors scale as :math:`1/n` where :math:`n` is the number of trajectories, and simultaneously does not take an excessive amount of time to run.  However, like many other options in QuTiP you are free to change the number of trajectories to fit your needs.  If we want to run 1000 trajectories in the above example, we can simply modify the call to ``mcsolve`` like:
 
 .. plot::
     :context: close-figs
 
-    data = mcsolve(H, psi0, times, [np.sqrt(0.1) * a], [a.dag() * a, sm.dag() * sm], ntraj=1000)
+    times = np.linspace(0.0, 10.0, 200)
+    psi0 = tensor(fock(2, 0), fock(10, 8))
+    a  = tensor(qeye(2), destroy(10))
+    sm = tensor(destroy(2), qeye(10))
+    H = 2*np.pi*a.dag()*a + 2*np.pi*sm.dag()*sm + 2*np.pi*0.25*(sm*a.dag() + sm.dag()*a)
+    data = mcsolve(H, psi0, times, [np.sqrt(0.1) * a], e_ops=[a.dag() * a, sm.dag() * sm])
+
+    plt.figure()
+    plt.plot((times[:-1] + times[1:])/2, data.photocurrent[0])
+    plt.title('Monte Carlo Photocurrent')
+    plt.xlabel('Time')
+    plt.ylabel('Photon detections')
+    plt.show()
+
+
+.. _monte-ntraj:
+
+Changing the Number of Trajectories
+-----------------------------------
 
-where we have added the keyword argument ``ntraj=1000`` at the end of the inputs.  Now, the Monte Carlo solver will calculate expectation values for both operators, ``a.dag() * a, sm.dag() * sm`` averaging over 1000 trajectories.  Sometimes one is also interested in seeing how the Monte Carlo trajectories converge to the master equation solution by calculating expectation values over a range of trajectory numbers.  If, for example, we want to average over 1, 10, 100, and 1000 trajectories, then we can input this into the solver using:
+As mentioned earlier, by default, the ``mcsolve`` function runs 500 trajectories.
```

Comment:
```suggestion
By default, the ``mcsolve`` function runs 500 trajectories.
```
I think the paragraph that mentions mcsolve has by default 500 trajectories was deleted.